driver-scala/src/main/scala/org/mongodb/scala/model/package.scala

```diff
@@ -19,7 +19,7 @@ package org.mongodb.scala
 import com.mongodb.annotations.{ Beta, Sealed }
 
 import scala.collection.JavaConverters._
-import com.mongodb.client.model.{ GeoNearOptions, MongoTimeUnit => JMongoTimeUnit, WindowOutputField }
+import com.mongodb.client.model.{ MongoTimeUnit => JMongoTimeUnit, SearchIndexType }
```

Comment:
What's the purpose of importing `SearchIndexType` here without introducing a `J`-prefixed alias? Currently, this import is quite confusing, because when one looks at `def apply(definition: Bson, indexType: SearchIndexType)`, it's difficult to tell whether the mentioned `SearchIndexType` is `com.mongodb.client.model.SearchIndexType` or `org.mongodb.scala.model.SearchIndexType`.

Reply:
This import was indeed unnecessary. I've removed it. Thanks for pointing it out!